src/WorkItems/WorkItems.csproj

```diff
@@ -4,6 +4,14 @@
     <RootNamespace>Microsoft.WorkItems</RootNamespace>
   </PropertyGroup>
 
+  <PropertyGroup Label="Packaging">
+    <!--
+    Without this property setting, the id of the NuGet package would match the
+    assembly name, "WorkItems".
```

Comment:
Let's actually change the name of the assembly to Microsoft.WorkItems. :)

Reply:
just updated! :)

Reply:
Merge it! :)

---
In reply to: [542029000](https://github.com/microsoft/sarif-sdk/pull/2180#discussion_r542029000) [](ancestors = 542029000)

Reply:
fixing the build and will merge it. thanks for reviewing it